templates/bundles/DemosPlanCoreBundle/DemosPlanProcedure/administration_edit_master.html.twig

```diff
@@ -276,7 +282,10 @@
                     <p class="lbl__hint u-ml-0_75">{{ 'explanation.customer.masterblueprint'|trans }}</p>
                 </label>
                 {% if isAnotherBlueprintSetAsCustomerMaster %}
-                    <p class="lbl__hint u-ml-0_75">{{ 'explanation.customer.masterblueprint.uncheck.existing'|trans }}</p>
+                    <p class="lbl__hint u-ml-0_75">{{ 'explanation.customer.masterblueprint.uncheck.existing'|trans({
```

Comment:
I'd display this either as a warning or an info (and I tend towards warning because the user is currently blocked from doing something)

```suggestion
                    <p class="flash flash-warning u-ml-0_75">{{ 'explanation.customer.masterblueprint.uncheck.existing'|trans({
```

Reply:
Dont you think that this would be too hidden?